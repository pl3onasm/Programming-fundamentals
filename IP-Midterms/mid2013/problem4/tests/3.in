5 6
5
4 2 1
0 2 1
1 2 1
2 0 5
3 2 1
