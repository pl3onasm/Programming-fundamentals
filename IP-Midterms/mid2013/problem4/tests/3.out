001000
001000
111110
001000
001000
